app/views/candidate_interface/start_page/show.html.erb

```diff
@@ -21,9 +21,12 @@
       <li>grade 4 (C) or above in GCSE science if you want to teach primary</li>
     </ul>
     <p class="govuk-body">
-      Before you start a teacher training course you’ll also need to confirm you have the health and physical capacity to start training.
+      Before you start a teacher training course, your training provider will usually send you a health questionnaire to complete. This confirms you have the fitness and physical capacity to start training.
     </p>
-    <h2 class="govuk-heading-m">If you don’t have a degree</h2>
+    <h2 class="govuk-heading-m">If you have a disability</h2>
+      <p class="govuk-body">It is against the law for training providers to discriminate against teacher training candidates with disabilities or special educational needs. <a href="https://getintoteaching.education.gov.uk/train-to-teach-with-a-disability">Learn more</a>.</p>
```

Comment:
This link needs to be a `govuk-link`. What we can do instead is this using a lil helper we have created:

```suggestion
      <p class="govuk-body">It is against the law for training providers to discriminate against teacher training candidates with disabilities or special educational needs. <%= govuk_link_to 'Learn more', 'https://getintoteaching.education.gov.uk/train-to-teach-with-a-disability' %>.</p>
```